tdrs-backend/docker-compose.yml

```diff
@@ -23,9 +23,13 @@ services:
       - DB_HOST=postgres
       - DJANGO_SETTINGS_MODULE=tdpservice.settings.local
       - DJANGO_CONFIGURATION=Local
-      - JWT_KEY=${JWT_KEY}
-      - JWT_CERT_TEST=${JWT_CERT_TEST}
-      - DJANGO_SU_NAME=${DJANGO_SU_NAME}
+      - JWT_KEY
+      - JWT_CERT_TEST
+      - DJANGO_SU_NAME
+      - AWS_ACCESS_KEY
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_BUCKET
+      - AWS_REGION_NAME
```

Comment:
looks like some env variables are introduced as part of this PR. a brief description of what these changes are about would be helpful :smiley:

Reply:
We got rid of the ${} after figuring out we did not need them.
`AWS_ACCESS_KEY` and `AWS_SECRET_ACESS_KEY` are credentials we use to sign the url, thats how we know this app has permission to be giving these presigned urls out.
`AWS_BUCKET` and `AWS_REGION_NAME` both end up in the resulting presigned url.

Reply:
Just to note, originally we did need the `${}` but with all of the changes, somewhere along the way they became unnecessary